dvc/fs/fsspec_wrapper.py

```diff
@@ -169,9 +172,16 @@ def _isdir(self, path_info):
             and entry["name"].endswith("/")
         )
 
-    def find(self, path_info, detail=False):
-        path = self._with_bucket(path_info)
-        files = self.fs.find(path, detail=detail)
+    def find(self, path_info, detail=False, prefix=None):
```

Comment:
It might be a nice idea (for future refactors of remotes) to pass the prefix in here (`0/`), and use the actual path (`bucket/cache/00/`) as the path_info (to be fully fsspec-compliant).

Reply:
Yeah, it would make more sense for us to do
```
walk("bucket/00/", prefix="0")
```
instead of the current
```
walk("bucket/00/0", prefix=True)
```